src/components/Footer.js

```diff
@@ -45,82 +54,111 @@ const BoldText = styled.p`
   letter-spacing: 2px;
 `;
 
+const BoldText2 = styled(BoldText)`
```

Comment:
Can we give this a more descriptive name? I cant tell where it goes with this name. Same for `BoldText`(1)